### PR TITLE
Shift to yarn

### PR DESCRIPTION
**Pull request by maheshschand**
This resolves issue #176 .

### CODE REVIEW

**Comment by maheshschand** [comment]
@aquibbaig Should I need to remove package-lock.json or not?

**Comment by ganeshpatro321** [comment]
@maheshschand  Please remove them.

**Comment by maheshschand** [comment]
But it showing conflict. Can you resolve it?

**Comment by Harkishen-Singh** [comment]
@maheshschand resolve the conflicts. You can do it easily in vscode

**Comment by maheshschand** [comment]
But the conflicting files is package-lock.json that I removed from my forked repo. Can you resolve it?

File: Makefile [hunk 1]
```diff
@@ -73,5 +73,4 @@ fix:
 	cd dashboard/v1.0/ && npm run tlint-fix
 
 lint:
-	golangci-lint run
-
+	golangci-lint run
```
Comment (aquibbaig) [inline Makefile:76]: Add trailing lines

**Comment by aquibbaig** [comment]
I think it's not been updated since. @maheshschand If it's difficult to resolve you can always close a PR and start a new one fetching the latest commit from "master".

**Comment by maheshschand** [comment]
OK. I am closing this pull request and opening a new one. Thanks for the review @aquibbaig

**Pull request closed without merging**